src/lib/prepareOas.ts

```diff
@@ -178,26 +183,23 @@ export default async function prepareOas(
   debug('👆👆👆👆👆 finished logging spec 👆👆👆👆👆');
   debug(`spec type: ${specType}`);
 
+  if (opts.title) {
+    debug(`renaming title field to ${opts.title}`);
+    api.info.title = opts.title;
+  }
+
   // No need to optional chain here since `info.version` is required to pass validation
   const specVersion: string = api.info.version;
   debug(`version in spec: ${specVersion}`);
 
-  let preparedSpec = '';
-
   if (['openapi', 'openapi:inspect', 'openapi:reduce'].includes(command)) {
-    preparedSpec = await oas.bundle().then(res => {
-      return JSON.stringify(res);
-    });
+    api = await oas.bundle();
 
     debug('spec bundled');
-  } else if (command === 'openapi:convert') {
-    // As `openapi:convert` is purely for converting a spec to OpenAPI we don't need to do any
-    // bundling work as those'll be handled in other commands.
-    preparedSpec = JSON.stringify(api);
   }
 
   return {
-    preparedSpec,
+    preparedSpec: JSON.stringify(api),
```

Comment:
Ah good idea. Even though it doesn't use it `openapi:validate` was getting only an empty string for this value.